tests/foreman/ui_airgun/test_role.py

```diff
@@ -0,0 +1,317 @@
+"""Test class for Roles UI
+
+:Requirement: Role
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: UI
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from airgun.session import Session
+from nailgun import entities
+from navmazing import NavigationTriesExceeded
+from pytest import raises
+
+from robottelo.datafactory import gen_string
+from robottelo.decorators import fixture, tier2, upgrade
+
+
+@fixture(scope='module')
+def module_org():
+    return entities.Organization().create()
+
+
+@fixture(scope='module')
+def module_loc():
+    return entities.Location().create()
+
+
+def test_positive_update(session):
```

Comment:
i'd rename it to test_positive_update_filter , as the test updates filter, not role.